shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rule/EncryptRule.java

```diff
@@ -294,6 +294,17 @@ public boolean isQueryWithCipherColumn(final String tableName) {
         return findEncryptTable(tableName).flatMap(EncryptTable::getQueryWithCipherColumn).orElse(queryWithCipherColumn);
     }
     
+    /**
+     * Judge whether column is support QueryWithCipherColumn or not.
+     *
+     * @param logicTable logic table name
+     * @param logicColumn logic column name
+     * @return whether column is support QueryWithCipherColumn or not
+     */
+    public boolean isQueryWithCipherColumn(final String logicTable, final String logicColumn) {
```

Comment:
yes, It can be, and I removed `isQueryWithCipherColumn(logicTable) `

```java
    public boolean isQueryWithCipherColumn(final String logicTable, final String logicColumn) {
        return findEncryptTable(logicTable).flatMap(encryptTable -> encryptTable.getQueryWithCipherColumn(logicColumn)).orElse(queryWithCipherColumn);
    }
```